portability-types-common/src/main/java/org/datatransferproject/types/common/models/photos/PhotoModel.java

```diff
@@ -22,8 +22,11 @@
 
 import java.util.Date;
 import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import org.datatransferproject.types.common.ImportableItem;
 
-public class PhotoModel {
+public class PhotoModel implements ImportableItem {
```

Comment:
(shouldn't block this PR) is it worth filing a bug to go update all the models to implement this?

Reply:
Good suggestion. Filed #1094